tests/boardfarm_plugins/README.md

```diff
@@ -0,0 +1,174 @@
+<!--
+SPDX-License-Identifier: BSD-2-Clause-Patent
+Copyright (c) 2020 the prplMesh contributors
+This code is subject to the terms of the BSD+Patent license.
+See LICENSE file for more details.
+-->
+
+### Prerequisites
+
+To obtain prplMesh and boardfarm sources a Google repo tool is used.
+Refer to its README for detailed info:
+https://gerrit.googlesource.com/git-repo/+/refs/heads/master/README.md
+
+### Boardfarm installation
+
+1. To obtain sources using previously mentioned repo tool run:
+```
+repo init -u https://github.com/prplfoundation/prplMesh-manifest.git
+repo sync
+repo forall -p -c 'git checkout $REPO_RREV'
```

Comment:
This is actually optional, it is perfectly fine to work in detached head (which is what repo is doing by default when its saying "no branch") - bottom line, you should add a comment here saying this is optional (nitpick)